playbooks/playbooks.json

```diff
@@ -0,0 +1,1296 @@
+{
+  "type": "workflow_collections",
+  "data": [
+    {
+      "@type": "WorkflowCollection",
+      "name": "Sample - Cybereason - 1.0.0",
+      "description": "Sample playbooks for \"Cybereason\" connector. If you are planning to use any of the sample playbooks in your environment, ensure that you clone those playbooks and move them to a different collection, since the sample playbook collection gets deleted during connector upgrade and delete.",
+      "visible": true,
+      "image": "/api/3/images/0223800a-57d0-4a3c-a980-cdf004626427",
+      "uuid": "812dd81f-e361-498f-a806-43e4a1430bba",
+      "id": 159,
+      "createUser": "/api/3/people/3451141c-bac6-467c-8d72-85e0fab569ce",
+      "createDate": 1624551093,
+      "modifyUser": "/api/3/people/3451141c-bac6-467c-8d72-85e0fab569ce",
+      "modifyDate": 1624551093,
+      "recordTags": [],
+      "workflows": [
+        {
+          "@type": "Workflow",
+          "triggerLimit": null,
+          "name": "Organization wide IP or Domain Blackist",
+          "aliasName": null,
+          "tag": null,
+          "description": "",
+          "isActive": true,
```

Comment:
sample collection playbooks should preferably be inactive unless they are use-case playbooks